File: 2026-08/src/day_26/prms_181902/Solution.java
package day_26.prms_181902;

class Solution {
    public int[] solution(String my_string) {
        int[] cnt = new int[52];
        for (char c : my_string.toCharArray()) {
            if (Character.isUpperCase(c)) {
                cnt[c - 'A']++;
            } else {
                cnt[c - 'a' + 26]++;
            }
        }

        return cnt;
    }
}

File: 2026-08/src/day_26/prms_181902/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

vector<int> solution(string my_string) {
    vector<int> cnt(52);
    for (char c : my_string) {
        if (isupper(c)) {
            cnt[c - 'A']++;
        } else {
            cnt[c - 'a' + 26]++;
        }
    }

    return cnt;
}
